docs/get-started/apps-integrations.md

```diff
@@ -188,31 +188,15 @@ The Log Analysis QuickStart app, created for new users of Sumo Logic, includes s
 
 Whether you're new to log management or plan to migrate from other products, the Log Analysis QuickStart app will bring you up to speed with the Sumo Logic search, visualization, and analytics capabilities.
 
-### Installation
+## Installing the Log Analysis QuickStart app
 
-To install the app:
+import AppInstallNoDataSourceV2 from '../reuse/apps/app-install-index-apps-v2.md';
```

Comment:
this app has source category so use `import AppInstall from '../reuse/apps/app-install-v2.md';`

Reply:
Have made the changes as suggested above.